src/transformers/models/llava_next_video/modeling_llava_next_video.py

```diff
@@ -1110,17 +1110,6 @@ def prepare_inputs_for_generation(
     ):
         # Overwritten -- extra custom processing
 
-        if input_ids is not None:
-            img_token_not_enough = (input_ids == self.config.image_token_index).sum(
-                1
-            ).max() < self.config.image_seq_length
-            video_token_not_enough = (input_ids == self.config.video_token_index).sum(
-                1
-            ).max() < self.config.video_seq_length
-            legacy_processing = (img_token_not_enough and pixel_values is not None) or (
-                video_token_not_enough and pixel_values_videos is not None
-            )
```

Comment:
glad that this is removed, never saw this being added (was not pinged on the PR!)